src/input.js

```diff
@@ -2,7 +2,7 @@ import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 
 import {AUTOCOMPLETE_CONTEXT} from './constants'
-import {compose, moveCursorToTheEnd} from './utils'
```

Comment:
Could you also remove this method from `utils`? It's not used anywhere else :smile: